### DISCUSSION TITLE
Getting Notable status on Algo Explorer

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#129] [General]
From [a correspondence](https://github.com/asastats/channel/issues/104#issuecomment-1149079029) with Algo Explorer officials:

- - - - - 

You can read about the requirements for each tier here: https://algoexplorer.io/reputation-system.

- - - - - 

We've opened a public discussion about (https://github.com/asastats/channel/discussions/129).

Thank you,

- - - - -

So, these are requirements:

### Notable status

Projects receiving widespread recognition by the community and maintaining a healthy reputation can become &quot;Notable&quot;.

These projects show strong interest from the community and have some of the following criteria: significant engagement metrics on social media and forums, growth in total value locked, the backing of reputable investors, and user engagement as tracked on chain, among other things. For this tier, complete information must be provided by the project.

- - - - -

**Comment by AlgoRhythMatic** [comment]
> significant engagement metrics on social media and forums

Would be interesting to try and find out the specific metrics required here - I don't see anything clear from the site.

> growth in total value locked, the backing of reputable investors

I think this may be heavier weight for consideration of transition from our current "Neutral" -> "Notable".

**Reply by ipaleka** [reply]
> > growth in total value locked, the backing of reputable investors
> 
> I think this may be heavier weight for consideration of transition from our current "Neutral" -> "Notable".

Well, our Poirot found some indirect connections with Borderless Capital... xD
